v2/cli/event_commands.go

```diff
@@ -0,0 +1,722 @@
+					Name:    flagYes,
+					Aliases: []string{"y"},
+					Usage:   "Non-interactively confirm cancellation",
+				},
+			},
+			Action: eventCancel,
+		},
+		{
+			Name:    "cancel-many",
+			Aliases: []string{"cm"},
+			Usage:   "Cancel multiple events without deleting them",
+			Description: "By default, only cancels events for the specified " +
+				"project with their worker in a PENDING phase",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:     flagProject,
+					Aliases:  []string{"p"},
+					Usage:    "Cancel events for the specified project only (required)",
+					Required: true,
+				},
+				&cli.BoolFlag{
+					Name:    flagRunning,
+					Aliases: []string{"r"},
+					Usage: "If set, will additionally abort and cancel events with " +
+						"their worker in a RUNNING phase",
+				},
+				&cli.BoolFlag{
+					Name:    flagYes,
+					Aliases: []string{"y"},
+					Usage:   "Non-interactively confirm cancellation",
+				},
+			},
+			Action: eventCancelMany,
+		},
+		{
+			Name:        "create",
+			Usage:       "Create a new event",
+			Description: "Creates a new event for the specified project",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:  flagPayload,
+					Usage: "The event payload",
+				},
+				&cli.StringFlag{
+					Name:  flagPayloadFile,
+					Usage: "The location of a file containing the event payload",
+				},
+				&cli.StringFlag{
+					Name:     flagProject,
+					Aliases:  []string{"p"},
+					Usage:    "Create an event for the specified project (required)",
+					Required: true,
+				},
+				&cli.StringFlag{
+					Name:    flagSource,
+					Aliases: []string{"s"},
+					Usage:   "Override the default event source",
+					Value:   "github.com/brigadecore/brigade/cli",
+				},
+				&cli.StringFlag{
+					Name:    flagType,
+					Aliases: []string{"t"},
+					Usage:   "Override the default event type",
+					Value:   "exec",
+				},
+			},
+			Action: eventCreate,
+		},
+		{
+			Name:  "delete",
+			Usage: "Delete a single event",
+			Description: "Unconditionally deletes (and aborts if applicable) a " +
+				"single event whose worker is in any phase",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:    flagID,
+					Aliases: []string{"i"},
+					Usage: "Delete (and abort if applicable) the specified event " +
+						"(required)",
+					Required: true,
+				},
+				&cli.BoolFlag{
+					Name:    flagYes,
+					Aliases: []string{"y"},
+					Usage:   "Non-interactively confirm deletion",
+				},
+			},
+			Action: eventDelete,
+		},
+		{
+			Name:    "delete-many",
+			Aliases: []string{"dm"},
+			Usage:   "Delete multiple events",
+			Description: "Deletes (and aborts if applicable) events for the " +
+				"specified project with their workers in the specified phase(s)",
+			Flags: []cli.Flag{
+				&cli.BoolFlag{
+					Name: flagAborted,
+					Usage: "If set, will delete events with their worker in an ABORTED " +
+						"phase; mutually exclusive with --any-phase and --terminal",
```

Comment:
Similarly, we could use formatted vars for job phases (in this case, replace "ABORTED" w/ `JobPhaseAborted`)

Reply:
Probably a good idea. I'll straighten that out tomorrow and open a separate PR to make similar changes to existing login, logout, and project management commands.

Reply:
Sounds good.  I am still slightly conflicted as it could definitely be argued that it reads better/more naturally as is... I'll leave it up to you whether you prefer the changes.

Reply:
> I am still slightly conflicted as it could definitely be argued that it reads better/more naturally as is...

That was my initial reasoning, tbh, but was willing to have my mind changed.

Using constants makes sense anywhere where there is a comparison or something, so that you can't accidentally get the logic wrong due to a mispelling, but here it's a different case.

And it's not as if `ABORTED` (for instance) is ever going to change to something else where having to update that in multiple places is a concern.

I guess I am back to thinking the way I was originally now that you've brought it up.

Maybe we leave it this way for now and agree to change it in the future if it becomes clear that we made the wrong choice?

Reply:
👍 Yea, I'm good with that.  I'm convinced we leave as is and revisit when/if it ever becomes an issue.